### ISSUE TITLE
Human Race(s?)

### ISSUE THREAD

**Issue by ihct** [ContributorCovenant/contributor_covenant#476]
Please correct me if I am wrong but isn't the word _"race"_ something strongly associated with biology? So by using the word _"race"_ isn't there an implication that we are talking about a biological classification here?

The literal translation also has found its way into the German translation, where the literal translation of the word is used (_Rasse_) which might leave an even more bitter taste thanks to history.

Would terms like _ethnic group_ or _ethnicity_ work as replacements?

Now what about skin color and other observable characteristics which are often used by others to make such a classification? Well, there is already _"personal appearance"_. I guess e.g. _"inherent physical traits"_ could be added here as well but not sure it is needed, because I hope it should be rather obvious that _"personal appearance"_ does not just refer one's hairstyle or wardrobe…

(BTW, not accepting this implied biological classification should not mean that one turns a blind eye to other people considering such a classification as sound [and mainly associating it with biology] and making discriminatory decisions based on it.)

**Comment by stephenburgess8** [comment]
I recommend closing this issue and PR because the person is suggesting a change without demonstrating they understand the fundamentals of the topic.

> The Census Bureau defines race as a person’s self-identification with one or more social groups.
https://www.census.gov/mso/www/training/pdf/race-ethnicity-onepager.pdf

Further considering the user account has a very low reputation due to almost no account history, the likelihood of it being a "troll" account is high.

**Comment by statgeek** [comment]
Old standard is Race statcan.gc.ca/eng/concepts/d…

New standard is ethnicity
www12.statcan.gc.ca/census-recense…

UN defaults to Ethnicity but notes that race and other terms are used. Also notes that usage definition should be clear https://unstats.un.org/unsd/demographic/sconcerns/popchar/popcharmethods.htm

I think replacing race with ethnicity makes sense.

**Comment by stephenburgess8** [comment]
@statgeek I think there may be a time and place for the discussion, which is one I think would benefit from a large amount of research, discussion, introspection and learning. However since OP did not provide that, I don't feel they did their due diligence. For that reason I would argue that OP or another should return will a well-considered argument citing several sources. Because of the highly political nature of this discussion and proposed action, it should not be something changed without a lot of support for doing so. I would rather see another issue and pull request that starting out with well-organized arguments and sources, sources, sources. OP does not have either.

Since the status quo is to use the word "race," the onus is on those who wish to change it to provide a convincing argument, not on the defence of including it (of which I think there is a strong one).

It's important to note that the terms "race" and "ethnicity" in the US have additional connotations and subtleties that arise from the reprehensible history of slavery in this country. In other countries, including Canada, this dynamic did not exist and so the meanings around the word "race" are different. It is very important to do justice to the civil rights movement in the US by using the correct language.

Still, this person did not provide a well-reasoned argument, provide sources, or seem to have anything but the most basic knowledge of the subject. In a procedural sense, I believe that should be sufficient to close this issue and PR until another issue can be made that present the argument more efficiently. One that acknowledges the complexity of the subject.

It's also important to acknowledge the low "reputation" of the OP. They seem to be highly fluent in how to make a Pull Request, yet their account shows no other activity.

To the contributors/collaborators of this project, thank you for maintaining this covenant. Much appreciation.

**Comment by stephenburgess8** [comment]
I believe that this PR is a form of subtle attack meant to underline the CC. Starting from [the history of attacks against the project and maintainer](https://philsturgeon.uk/2016/09/15/codes-of-conduct-maybe-theyre-not-so-bad/), I believe this should be flagged as a likely troll attempt. It's possible it's not, but if it walks like a duck, swims like a duck, and quacks like a duck, it's probably a duck. When I search for "code of conduct sjw" this repository is the first result. Even the alt-right site "BreidBard" has attacked CoCs. For me, creating a researched issue around a very sensitive issue from an account with no history would be sufficient cause for suspension of the account and closing of the issue.

It's important not to ask those who experience institutional discrimination to be the ones to educate everyone else about this issue. My motivation in speaking up is to reduce the amount of work a person of color may need to do in this conversation. However I would be very curious to hear from someone who has done extensive research on this topic.

People should not be discriminated against based on their race. Race should not be removed from this list because people have been and are discriminated against on basis of their race. In the US, race was the line along which there were historical injustices, it was the line along which segregation occurred, and it's the line along which institutional discrimination persists to this day. Ethnicity was not the line against those injustices primarily occurred. It's more precise to say that race was.

That's all not to say that discrimination can't occur against people based on their ethnicity. It can and it does. For that reason it may be appropriate to include ethnicity in the list, but not if it replaces race. Either way, I think OP's account should be flagged for behavior that is typical of accounts created for the purpose of trolling.

Thanks for your time on the Contributor Covenant.

**Comment by egircys** [comment]
@ihct thank you for addressing this issue. Would maybe `ethnic origin` be more suitable than `ethnicity`? As commented in https://github.com/symfony/symfony-docs/pull/9394#discussion_r172696659

**Comment by CoralineAda** [comment]
This is really helpful reading: https://www.diffen.com/difference/Ethnicity_vs_Race

I think that there may be a difference in the way that the words race and ethnicity are used in the US vs other countries. The US has for centuries struggled with issues of race-based violence and discrimination. Race is the word used to frame that conflict.

Ethnicities, I feel, are more about cultural origins. I'm an ethnic German, but my race is "white". I don't enjoy privilege because of my ethnicity, but I do have privilege as a result of my racial identification.

My feeling is that, at the risk of being US-centric, the inclusion of both words is important, because in practice they don't mean the same thing.

**Comment by ShadowGoat** [comment]
this is just as cancerous as everything else. Also, being white doesn't give you privilege.

**Comment by retrosight** [comment]
@CoralineAda I believe your reply immediately above would be a good candidate for the FAQ, to explain the difference. Happy to create a pull request to add -- if that would be appropriate.

**Comment by CoralineAda** [comment]
Please do! That's a great idea.

**Comment by tikabass** [comment]
Adding 'race', 'ethnicity', 'cultural origin', are all attempts to classify people.  As is 'political views', 'language', 'social group'.  This makes pretty much all of the CoC discriminatory by design...  As to the 'enforcement' provisions, I thought we had gotten rid of that way of thinking at the end of WWII.

**Comment by Uzlopak** [comment]
In the German constitution we have in article 3: "no one shall be discriminated because of his [...]race[...]". When the constitution was written in 1949 no one thought about it being wrong and like everything it got interpreted correctly as "no one shall be discriminated based on racist views.

**Comment by seschu01** [comment]
@CoralineAda "The US has for centuries struggled with issues of race-based violence and discrimination. Race is the word used to frame that conflict."

Really? No other country has ever had this issue? It is isolated to the US? Please, if you are going to run this thing try to abstain from generalities.

**Comment by stephenburgess8** [comment]
@CoralineAda Thank you for hosting a discussion on this topic. I strongly feel that race is an entirely appropriate addition to this list. I think there's a reason the term is "racism" and not "ethnicitism" which points to race being the correct word. The common discussion about discrimination is discrimination against people of one race or another, not against one ethnicity or another. That's not to say that someone couldn't be discriminated against based on their ethnicity. In this case the word "race" is acting as an umbrella term.

Thank you for pioneering the contributor covenant and really broadening discussions as about codes of conduct and agreed upon standards of behavior at conferences and in other venues.

**Comment by SaschaMann** [comment]
Sorry to revive this seemingly stale discussion but I was pointed to this repo because [Exercism](https://exercism.org/)'s CoC is based on this code of conduct and that this should be changed "upstream" first.

---

About the FAQ addition mentioned above:

> Race is the grouping of humans based on shared physical or social qualities into categories generally viewed as distinct by society and usually resulting from genetic ancestry.

This feeds into the idea that different human races exist which has long been disproven. The FAQ should be updated to underline that any such grouping is inherently unwelcome and archaic.

I understand that, in particular in the US, there are different understandings of the term that try to revive this idea and categorisation system, but given that this CoC is used in rather international contexts, the more general understanding should outweigh this.

(As a side note, this [appears to be a 1:1 quote](https://en.wikipedia.org/wiki/Race_(human_categorization)#cite_note-Barnshaw-1) and the source should be stated in the FAQ in case it is indeed one)

---

> I think there's a reason the term is "racism" and not "ethnicitism" which points to race being the correct word.

Racism is the creation of such groupings into "races" (and as a consequence, discrimination based on them). The CoC implies that such "races" exist since otherwise one could not discriminate against someone based on a "race". However it is obviously possible to first create such nonsensical groupings and then discriminate based on them. This distinction should be highlighted by the code of conduct.

---

I would support removal or replacement of the term "race" since as it is the code of conduct itself implies that there are different human races in its current form. However, that does not seem like a popular amendment based on this issue, so updating the FAQ would be a welcome first step.

---

> In the German constitution we have in article 3: "no one shall be discriminated because of his [...]race[...]". When the constitution was written in 1949 no one thought about it being wrong and like everything it got interpreted correctly as "no one shall be discriminated based on racist views.

There is a lot of discussion about fixing this historic mistake. Today the (likely) future government parties have [announced](https://cdn.businessinsider.de/wp-content/uploads/2021/11/Koalitionsvertrag-2021-2025.pdf#page=120) to fix it and remove the term entirely from the constitution.

**Comment by Uzlopak** [comment]
@SaschaMann 
Exactly. I would prefer something like article 14 ehrc where it says 

```
sex, race, colour, language, religion, political or other opinion, national or social origin, association with a national minority, property, birth or other status.
```

Obviously it contains the word race and allows differing "opinions", which btw differing political opinions are not allowed by the convention. 
Even chinese, who were second born children and were highly discriminated by the Chinese government were subsumed as a distinct group of social origin. 

You could always subsume race under color and/or social origin, if you are somebody who thinks race is a social construct.
